enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/service/CommonOtpService.java

```diff
@@ -146,8 +158,8 @@ private void handleFailedOtpVerification(OnboardingProcessEntity process, OwnerI
             otp.setTimestampFailed(ownerId.getTimestamp());
             onboardingOtpRepository.save(otp);
 
-            // Onboarding process is failed, update it
-            process = processLimitService.failProcess(process, OnboardingOtpEntity.ERROR_MAX_FAILED_ATTEMPTS, ErrorOrigin.PROCESS_LIMIT_CHECK);
+            // Fail process or identity verification based on OTP type
+            handleOtpMaxFailedAttempts(process, otp, ownerId);
```

Comment:
Nit: Here we have three information, an inline comment, a method name and javadoc. I guess we may omit the inline comment and rename a method, if you wish, for example to `failProcessOrIdentityVerification(process, otp, ownerId)`.

Reply:
Ok.